src/locale/locales/de/messages.po

```diff
@@ -922,30 +758,26 @@ msgstr "Bücher"
 
 #: src/components/FeedInterstitials.tsx:281
 msgid "Browse more accounts on the Explore page"
-msgstr ""
+msgstr "Stöbere auf der Seite „Explore” nach weiteren Konten"
```

Comment:
the page is probably not called `Explore` in german

Reply:
I'll have to double-check this, my understanding is that it's not been translated (like `Following`)